Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,23 @@
 import React from 'react';
 import './Header.css';
+import HeaderContent from "./HeaderContent";
+import moment from 'moment';
```

Comment:
This package was either not installed or wasn't installed from the right directory. How can i tell? Because this package is missing from the dependencies list in the `package.json` file. But how did it work locally? Locally, if the package was installed in a parent directory it will work.